modules/nodes-containers-downward-api-container-values-envars.adoc

```diff
@@ -45,15 +45,18 @@ spec:
             fieldRef:
               fieldPath: metadata.namespace
   restartPolicy: Never
+# ...
 ----
 
-. Create the pod from the `pod.yaml` file:
+.. Create the pod from the `pod.yaml` file:
 +
 [source,terminal]
 ----
 $ oc create -f pod.yaml
 ----
 
+.Verification
```

Comment:
I don't think this should have a number, if it's the only verification criteria. See RH supplementary style guide: https://redhat-documentation.github.io/supplementary-style-guide/#single-step-procedures